modules/core/src/main/java/org/apache/ignite/internal/processors/pool/PoolProcessor.java

```diff
@@ -461,6 +461,9 @@ public PoolProcessor(GridKernalContext ctx) {
                 oomeHnd
             );
 
```

Comment:
Please remove empty line before  the end of if-block

Reply:
Fixed